networking/hardware_networks/about-sriov.adoc

```diff
@@ -59,5 +59,6 @@ include::modules/nw-sriov-example-vf-function-in-pod.adoc[leveloffset=+2]
 * xref:../../networking/hardware_networks/installing-sriov-operator.adoc#installing-sriov-operator[Installing the SR-IOV Network Operator]
 * Optional: xref:../../networking/hardware_networks/configuring-sriov-operator.adoc#configuring-sriov-operator[Configuring the SR-IOV Network Operator]
 * xref:../../networking/hardware_networks/configuring-sriov-device.adoc#configuring-sriov-device[Configuring an SR-IOV network device]
+* xref:../../virt/virtual_machines/vm_networking/virt-configuring-sriov-device-for-vms.adoc#virt-configuring-sriov-device-for-vms[Configuring an SR-IOV network device for virtual machines]
```

Comment:
I think it's fair to add it as a next step, but I might add a clarification, like "If you use {VirtProductName}, xref:.."
Otherwise, it seems required. (I think this is better than labeling it as optional if it's actually required for a specific use case.)